example/src/Examples/API/Touch.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useCallback, useRef } from "react";
+import { StyleSheet, View } from "react-native";
+import type { SkColor, TouchInfo } from "@shopify/react-native-skia";
+import {
+  Fill,
+  Canvas,
+  Drawing,
+  Skia,
+  PaintStyle,
+  usePaint,
+  TouchType,
+} from "@shopify/react-native-skia";
+import type { DrawingContext } from "@shopify/react-native-skia/src/renderer/DrawingContext";
+
+import { Title } from "./components/Title";
+
+const Colors = [
+  "#2D4CD2",
+  "#3CF2B5",
+  "#A80DD8",
+  "#36B6D9",
+  "#37FF5E",
+  "#CF0CAA",
+  "#AFF12D",
+  "#D35127",
+  "#D01252",
+  "#DABC2D",
+  "#5819D7",
+] as const;
+
+export const Touch = () => {
+  const paint = usePaint((p) => {
+    p.setStyle(PaintStyle.Stroke);
+    p.setStrokeWidth(8);
+  });
+  const currentTouches = useRef<Array<TouchInfo & { color: SkColor }>>([]);
+
+  const handleTouches = useCallback((touchInfo: Array<Array<TouchInfo>>) => {
```

Comment:
Why is this not using the regular `useValue`/`useTouchHandler` API?

Reply:
Because the regular touch handler does not support multitouch...